actions.yaml

```diff
@@ -0,0 +1,2 @@
+get-ca-cert:
```

Comment:
I'd recommend using `certificate` instead of `cert`. The charm itself is called "self-signed-certificates" not "self-signed-certs" and standardising around naming sounds reasonable. 

This applies at a couple of places.

Reply:
Done.